subprojects/groovy-json/src/main/java/groovy/json/JsonOutput.java

```diff
@@ -119,12 +119,24 @@ public static String toJson(String s) {
      * @return a formatted date in the form of a string
      */
     public static String toJson(Date date) {
+        dateFormat.setTimeZone(TimeZone.getTimeZone(DEFAULT_TIMEZONE));
```

Comment:
See note above, probably should new up a `SimpleDateFormat` here, might be better to provide a way to override the JSON_DATE_FORMAT that is used and construct a new format object using either the provide format string and fall back to JSON_DATE_FORMAT if not provided.